tests/puppeteer/campaigns/sanity/02_productsBO/02_CRUDStandardProductInBO.js

```diff
@@ -8,8 +8,8 @@ const loginCommon = require('@commonTests/loginBO');
 const LoginPage = require('@pages/BO/login');
 const DashboardPage = require('@pages/BO/dashboard');
 const BOBasePage = require('@pages/BO/BObasePage');
-const ProductsPage = require('@pages/BO/products');
-const AddProductPage = require('@pages/BO/addProduct');
+const ProductsPage = require('@pages/BO/catalog/products/products');
```

Comment:
Same
```suggestion
const ProductsPage = require('@pages/BO/catalog/products');
```

Reply:
we are sorting pages by actual path in BO : 
So `@pages/BO/catalog/products/products` means `list of products` in Menu Products. with this way, we can have `@pages/BO/catalog/products/addProduct;` in same directory

Same for all suggestions

Reply:
Ohhh. What if the Core change paths and title, like we did few months ago for module manager? And AFAIK, it will be done again in few months for the same part.
Why don't you do it by features?

`@pages/BO/catalog/products` => `/pages/BO/catalog/products/index.js` => Product listing

Reply:
OK for me, for renaming `products/products.js` to `products/index.js`

Reply:
So that means we need to make another level of directory listing, since we can't have a `products.js` and a `addProduct.js` in the same directory...
Not sure if that's an improvement !

Reply:
@SimonGrn `/pages/BO/catalog/products/add.js` means you're in catalog products, and want to add one. Independent from bo architecture.